tests/conftest.py

```diff
@@ -26,6 +27,23 @@ def image_factory():
     return img_factories.ImageFactory()
 
 
+@pytest.fixture
+def gis_point():
+    return Point(13.397788148643649, 52.52958586909979)
```

Comment:
nit-picking, but can you stick to same naming either `gis_point` or `geos_point` as you have been naming it in the rest of the code?